site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/FhirContextWrapper.java

```diff
@@ -44,6 +45,21 @@ public Bundle executeTrasaction(String uri, Bundle input) {
                     .withBundle(input).execute();
     }
 
+    /**
+     * Execute a bundle transaction to a FHIR endpoint
+     * @param uri the FHIR endpoint URI
+     * @param name of the organization to search.
+     * @return Organization searched by name.
+     */
+    public Bundle executeSearchByName(String uri, String name) {
```

Comment:
Since this handles Orgs specifically, you should note that in the method name.
Also, seems like "get" is more appropriate than execute.

Reply:
Agreed to keep same method name